### ISSUE TITLE
False error after opening and closing PHP-tag

### ISSUE THREAD

**Issue by wedi** [WordPress-Coding-Standards/WordPress-Coding-Standards#262]
If you have to print out a large chunk of HTML it is convinient to use a closing and opening PHP tag.
Unfortunately the following code produces an indentation error as outlined below:

```php
<?php
class Foo {
    function bar1() {
        ?>Output some HTML
        <?php
        }
    function bar2() {  //  indented incorrectly; expected at least 2 spaces, found 1'
        echo 'This line is said to be indented incorrectly; expected at least 3 spaces, found 2';
        echo 'This one, too.';
        echo 'And it doesn\'t end here. It\'s for the rest of the file.';
    }
}
``` 

If you move the colsing php tag to the beginning of the line the error is gone, but phpcs complains about the `<?php` not being indented correctly.

**Comment by JDGrimes** [comment]
Fixed in 0.3.0. Closing.
